### PR TITLE
add troubleshooting for loadbalancer restores

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

See issue #264

### CODE REVIEW

**Comment by Bradamant3** [comment]
Makes sense -- content moved.

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -32,6 +34,11 @@ kubectl edit deployment/ark -n heptio-ark
 ...
 ```
 
+## Known issue with restoring LoadBalancer Service
+
+Because of how Kubernetes handles Service objects of `type=LoadBalancer`, you might encounter an issue with changed values for Service UIDs. If the DNS CNAME for your application points to the DNS name of your cloud load balancer, you'll need to update the CNAME pointer when you perform an Ark restore.
```
Comment (skriss) [inline docs/troubleshooting.md]: "...objects of `type=LoadBalancer`, when restoring them, you might encounter an issue with changed DNS names for external load balancers.  Kubernetes automatically generates the name of the cloud resource based on the `Service` UID, which will be different when restored, resulting in a different name for the cloud load balancer.  If the DNS CNAME for..."

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -32,6 +34,11 @@ kubectl edit deployment/ark -n heptio-ark
...
+
+Alternatively, if your cloud provider supports an external load balancer, you might be able to use the Service's `spec.loadBalancerIP` field to keep connections valid. See [the Kubernetes documentation about Services of Type LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). For more information, see [the documentation about how to create an External Load Balancer while preserving the client source IP](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip).
```
Comment (skriss) [inline docs/troubleshooting.md]: "Alternatively, you might be able to use the Service's `spec.loadBalancerIP` field if your cloud provider supports it to keep connections valid."

Also, that last sentence/link is not relevant here so can be removed.

File: docs/troubleshooting.md (hunk 1 above)
Comment (skriss) [reply docs/troubleshooting.md]: It changes when you run the restore -- when the `Service` obj is applied, it gets a new UID, and that's used to name the corresponding load balancer that gets created

**Comment by skriss** [comment]
@Bradamant3 I'll merge once you squash

**Pull request merged**